src/meta/meta_ckpt.c

```diff
@@ -471,19 +471,53 @@ __ckpt_valid_blk_mods(WT_SESSION_IMPL *session, WT_CKPT *ckpt, bool rename)
     return (0);
 }
 
+/*
+ * __ckpt_copy_blk_mods --
+ *     Copy the block mods from a source checkpoint to the destination checkpoint.
+ */
+static int
+__ckpt_copy_blk_mods(WT_SESSION_IMPL *session, WT_CKPT *src_ckpt, WT_CKPT *dst_ckpt)
+{
+    uint64_t i;
```

Comment:
`WT_BLKINCR_MAX` is 2, probably don't need 8B here (but it's not keeping me up nights, either).